src/arch/8051/arch.mk

```diff
@@ -14,7 +14,7 @@ sim: $(BUILD)/ec.rom
 # Convert from Intel Hex file to binary file
 $(BUILD)/ec.rom: $(BUILD)/ec.ihx
 	@mkdir -p $(@D)
-	makebin -s $(CODE_SIZE) -p < $< > $@
```

Comment:
`-s $(CODE_SIZE)` did absolutely nothing because it was immediately followed by `-p`.

If we want to pad the ROM, `--pad-to=$(CODE_SIZE)` can be used. I would recommend padding to the full ROM size though (128k for ITE, 160k(?) for ENE).